README.md

```diff
@@ -76,6 +76,8 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [Android Developer Roadmap](https://roadmap.sh/android)
 - [Android cheat sheet](http://alvinalexander.com/android/android-cheat-sheet-main-concepts/)
 - [Android Tips & Tricks](https://github.com/nisrulz/android-tips-tricks)
+- [Android Developer Certification Exam Guide](https://developers.google.com/certification/associate-android-developer/study-guide)
```

Comment:
Replace with the actual title of the page.

Reply:
@shahednasser this is valid you can checkout the link for confirmation

Reply:
I think you mean the link below. I checked the page, but I still prefer to remove it.

Reply:
@shahednasser can I atleast know the reason why it cannot be merged ?